api-reference/listing/post.mdx

```diff
@@ -422,16 +402,14 @@ curl --location --request POST 'https://api.travtus.com/listings/' \
   "is_furnished": true,
   "lease_terms": [
     {
-      "identifier": "listing-1-lease-term-1",
-      "group": "group-id-1",
       "deposit": 1400.00,
       "length_in_months": 12,
       "rent": 700.00
     }
   ],
   "active_date": "2023-01-17",
   "move_in_date": "2023-01-17",
-  "sqft": 40,
+  "sqft": 40.00,
```

Comment:
Is there a reason for the decimal point? Is it required?

Reply:
The type for the schema object in the knowledge service is float, so I wanted to reflect that in the example